tests/GraphQL/AddElementToAreaMutationTest.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace DNADesign\Elemental\Tests\GraphQL;
+
+use InvalidArgumentException;
+use SilverStripe\Dev\SapphireTest;
+use GraphQL\Type\Definition\ResolveInfo;
+use DNADesign\Elemental\Models\ElementalArea;
+use DNADesign\Elemental\Tests\Src\TestElement;
+use DNADesign\Elemental\GraphQL\AddElementToAreaMutation;
+
+class AddElementToAreaMutationTest extends SapphireTest
+{
+    protected static $fixture_file = 'AddElementToAreaMutationTest.yml';
```

Comment:
Your fixture file is empty - I realise this is because you've reported having problems with `$usesDatabase = true`, but maybe document it. I've never had a problem with that property, so at some point somebody is going to wonder why this is here and remove it =)

Reply:
I think if I combine it with the previous suggestion we might knock two off the list with one change :)